vtex/checkout-ui-tests
# Update monitoring entrypoint to accept CLI arguments

#### What is the purpose of this pull request?

Updates the `src/monitoring/index.js` (which is accessible through the `yarn test` script) to accept CLI arguments to customize the test run to narrow the amount of tests that will be executed.

#### What problem is this solving?

When the need of running a subset of the tests files on a specific environment, on a specific account arises, we don't have many options to do that in this repository (changing the scripts in the `package.json` to be able to just change the workspace that the tests will run in is really bad and error prone). So, I transformed the entrypoint script that runs our tests to accept CLI arguments that will change the options passed down to Cypress in order for it to be a little more flexible. You can access all the available options by running `yarn test --help`.

#### How should this be manually tested?

Run `yarn test -a vtexgame1` and see that the tests will only execute in the `vtexgame1` account :satisfaction:

#### Types of changes
